dev-client/src/translations/en.json

```diff
@@ -434,6 +434,12 @@
       "photos": "Photos",
       "notes": "Notes"
     },
+    "verticalCracking_description": "Vertical cracks may appear on the soil surface when some soils experience changes in moisture or temperature. The cracks may be permanent, or they may disappear after some time.\n\nVertical cracks are at least 5 mm (3/16”) wide and 25 cm (10”) deep. You should be able to place a pencil inside the cracks.",
```

Comment:
issue: use either cm or in depending on what units the user has selected. Add additional keys to the JSON and pass them in as parameters.

Reply:
@CourtneyLee333 do you want text in tooltips/info sheets to respond to the user's preferred units?

Reply:
Would that mean calculations need to get made, since inch and cm, yard and meter, etc aren't exactly the same? @ltseng do you have a thought about this?

Reply:
for info sheets we would just be choosing different text depending on the units, so we can choose when it's appropriate to write e.g. 3/4" vs 0.7567234571248" etc. in this case rather than the text:
> Vertical cracks are at least 5 mm (3/16”) wide and 25 cm (10”) deep.

The user would see only one of the following depending on their preferred units:
> Vertical cracks are at least 5 mm wide and 25 cm deep.
> Vertical cracks are at least 3/16” wide and 10” deep.

Reply:
I think that sounds ideal!